pkg/processresetqueue/reset.go

```diff
@@ -68,51 +67,27 @@ func Reset(input *ResetInput) (*ResetOutput, error) {
 		// result and move on to the next account.
 		for _, message := range messages.Messages {
 			// Assumes the Message's Body to just contain the Account's ID to Reset
-			accountID := *message.Body
 			result := ResetResult{
 				BuildTrigger:    false,
 				MessageDeletion: false,
 			}
 
-			// Get the Account from the Database
-			account, err := input.DbSvc.GetAccount(accountID)
-			if err != nil {
-				failTriggerResetOnAccount(&output, result, accountID,
+			acct := &account.Account{}
```

Comment:
Using the new account model as this will parse things into the ARNs and make the next set of lines of code unnecessary .